shared/whatYouNeed/raspberrypi.md

```diff
@@ -3,7 +3,7 @@
 * A 4GB or larger SD card. All the Raspberry Pi, except older model Bs, uses a microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
 * An ethernet cable or WiFi adapter (not needed for the Zero W). Check our [list of supported wifi adapters][wifiAdapters].
 * A micro USB cable.
-* **[Optional]** A [2A USB micro power supply][psu]
+* **[Optional]** A [2A USB micro power supply][psu].
```

Comment:
nit: It's a "micro USB power supply" rather than a "USB micro power supply"

Reply:
Also, why do we need "A micro USB cable."?
AFAICT the requirement is _either_ for a micro USB power supply like [this](https://www.amazon.co.uk/Micro-Supply-Charger-Raspberry-Pi/dp/B00AUKR4EU) (which has a moulded microUSB plug) **OR** a USB power supply like [this](https://www.amazon.co.uk/Technoplay-ltd-Mains-Charger-Charge/dp/B001SGXGGA) and a micro USB cable?